localisation/ko/README.md

```diff
@@ -0,0 +1,100 @@
+# GitHub Copilot Vibe 코딩 워크샵
+
+![GitHub Copilot - Ghiblifiled](./images/ghcp.jpg)
```

Comment:
This image path should point to the original one, which is `../../images/ghcp.jpg`